pyoos/parsers/ioos/get_observation.py

```diff
@@ -27,13 +29,13 @@ def __new__(cls, element):
 
         if version == "1.0":
             from pyoos.parsers.ioos.one.get_observation import GetObservation as GO10
-            return super(IoosGetObservation, cls).__new__(GO10, element=root)
+            return super(IoosGetObservation, cls).__new__(GO10)
```

Comment:
the `__new__` method always confuses me.  Does this mean it gets the arguments from the call automatically?  worried about losing the `element` param here.